src/Http/CallbackStream.php

```diff
@@ -40,7 +40,8 @@ public function getContents(): string
     {
         $callback = $this->detach();
         $result = '';
-        if (is_callable($callback)) {
+        /** @psalm-suppress TypeDoesNotContainType */
+        if ($callback !== null) {
```

Comment:
Why is this needed?

Reply:
The return type of `detach()` is `?callable` so checking `!== null` is cheaper hence I made the change. Pslam gave some silly error like `Type null for $callback is always null` which I ignored.

Reply:
It probably doesn't consider callable a native type and discards it. Probably a bug.